global.net8.json

```diff
@@ -0,0 +1,12 @@
+{
+  "tools": {
+    "dotnet": "8.0.100"
+  },
+  "msbuild-sdks": {
+    "Microsoft.DotNet.Arcade.Sdk": "9.0.0-beta.23580.7",
```

Comment:
Shouldn't we be targeting the 8.0 version of Arcade here? Otherwise we might end up in the same boat where we need the 9.0 SDK to build some part of Arcade.

Reply:
Updated to match the versions specified in the dotnet/runtime release/8.0-staging global.json.